homeassistant/components/synology_dsm/__init__.py

```diff
@@ -64,7 +66,8 @@
     {
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_PORT): cv.port,
-        vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+        vol.Optional(CONF_SSL, default=DEFAULT_USE_SSL): cv.boolean,
+        vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
```

Comment:
We no longer allow changes to YAML configuration for integration that communicate with a device or service.

For more information see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md>

This integration needs to be adjusted to use a configuration flow. To get started on that, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
The integration already uses a config flow, see:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/synology_dsm/config_flow.py

Reply:
Sorry my bad 😅